source/server/config/network/http_connection_manager.cc

```diff
@@ -43,12 +42,25 @@ NetworkFilterFactoryCb HttpConnectionManagerFilterConfigFactory::createFilterFac
                                                                       context.threadLocal());
           });
 
+  std::shared_ptr<Router::RouteConfigProviderManager> route_config_provider_manager =
+      context.singletonManager().getTyped<Router::RouteConfigProviderManager>(
+          route_config_provider_manager_singleton_name, [&context] {
```

Comment:
We should have a macro to get this name, rather than assuming folks know to suffix `_singleton_name`.